### PR TITLE
Added feature for new dl

### PR DESCRIPTION
**Pull request by sameepsi**

### CODE REVIEW

**Review by totop716 — APPROVED** [review]
Looks good. Pushed a small update on UI.

File: src/utils/index.ts [hunk 1]
```diff
@@ -1896,9 +1898,12 @@ export function useLairDQUICKAPY(lair?: LairInfo) {
       daysCurrentYear) /
     dQUICKPrice;
   if (!dQUICKAPR) return '0';
-  return Number(
-    (Math.pow(1 + dQUICKAPR / daysCurrentYear, daysCurrentYear) - 1) * 100,
-  ).toLocaleString();
+  const temp = Math.pow(1 + dQUICKAPR / daysCurrentYear, daysCurrentYear) - 1;
+  if (temp > 100) {
+    return '> 10000';
```
Comment (RitzyDevBox) [inline src/utils/index.ts:1903]: what's the reasoning behind this? we don't do this for LP with apy that large?

Comment (sameepsi) [reply src/utils/index.ts:1903]: The reasoning is that APY might be in millions which will look bad in the UI. On old UI we already do this for high APYs. I am not sure about beta

Comment (sameepsi) [reply src/utils/index.ts:1903]: The reasoning is that APY might be in millions which will look bad in the UI. On old UI we already do this for high APYs. I am not sure about beta

**Pull request merged**